Sources/UtiliKit/Container/Transitioning/ContainerViewControllerInteractiveAnimatedTransitioning.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ContainerViewControllerInteractiveAnimatedTransitioning.swift
```

Comment:
Should we go ahead and name this file `ContainerViewControllerInteractiveTransitioning.swift` to be consistent with the `ContainerViewControllerInteractiveTransitioning` protocol defined below?